src/plugins/commit-message/index.js

```diff
@@ -30,13 +65,30 @@ const EXCLUDED_REPOSITORY_NAMES = new Set([
  */
 function checkCommitMessage(message) {
     const commitTitle = message.split(/\r?\n/)[0];
+    const errCode = [];
 
     if (message.startsWith("Revert \"")) {
-        return true;
+        return [true, []];
     }
 
+    let isValid = true;
```

Comment:
Do we still need this variable to track whether the message passed validation or not? Is there anything stopping us from checking `errCode.length === 0` to see if the message is valid or not?